applications/dashboard/views/settings/registration.php

```diff
@@ -73,9 +73,12 @@
 <div id="InvitationTarget" class="form-group">
     <div class="label-wrap">
         <?php echo $this->Form->label('Invitation target', 'Garden.Registration.InviteTarget'); ?>
+        <div class="info">
+            <?php echo t('Users will be redirected to this URL after accepting an invitation. It can be a full URL or a path to redirect within the site.'); ?>
```

Comment:
When translations are this long we usually shorten the translation code so that the translation file is cleaner. In this case choosing the first sentence might be a good idea.